liblouis/compileTranslationTable.c

```diff
@@ -3820,11 +3821,15 @@ compileRule (FileInfo * nested)
 		case CTO_Match:
 		{
 			CharsString ptn_before, ptn_after;
-			widechar patterns[27720];
 			TranslationTableOffset offset;
 			int len, mrk;
 
-				memset(patterns, 0xffff, sizeof(patterns));
+			size_t patternsByteSize = sizeof(*patterns) * 27720;
+			patterns = (widechar*) malloc(patternsByteSize);
+			if(!patterns)
+				outOfMemory();
+			memset(patterns, 0xffff, patternsByteSize);
```

Comment:
Why aren't you using `calloc` here (instead of `malloc` and `memset`)? Other places in the code seem to use `calloc`